### ISSUE TITLE
ZTag generic modbus manufacturer

### ISSUE THREAD

**Issue by zakird** [zmap/ztag#24]
We should create an annotation that pulls out manufacturer and model from any modbus scan and generically assigns it.

**Comment by dadrian** [comment]
I don't think we should do this until we can guarantee that this will be "stomped" by vendor-specific tags.

**Comment by zakird** [comment]
Yeah. :| I thought about that too. The good news is that we could actually do this in ZTag since it's all in a single protocol.
